### DISCUSSION TITLE
Reload not working for Month view

### DISCUSSION THREAD

**Discussion by jenithraja** [kvyatkovskys/KVKCalendar#279] [Q&A]
When i try to add an event for Aug 12 in admin end then i call api and first time the dots showing in Aug 12. When i try to add one more different event on same day the calender view is not updated with more than one dot.

When i kill the app and come back again now i could see two dots on same day.

Please help me how to reload the view when the app is in foreground.

**Comment by kvyatkovskys** [comment]
Do you call the `calendar.reloadData()` function?

**Reply by jenithraja** [reply]
Yes, i have called the calender.reloadData() but no luck.
Please suggest, once the dots are added i cannot able to remove the dots or i cannot add new dots when i tried to go back from next or previous month.

**Reply by jenithraja** [reply]
Awaiting for your reply?

**Reply by kvyatkovskys** [reply]
I'll check your issue on this weekend

**Reply by jenithraja** [reply]
Have you checked?

**Reply by kvyatkovskys** [reply]
yep, I checked... works fine for me
